lib/shopify_app/sessions_concern.rb

```diff
@@ -87,5 +88,16 @@ def return_address
       session.delete(:return_to) || main_app.root_url
     end
 
+    def perform_after_authenticate_job
+      return unless ShopifyApp.configuration.after_authenticate_job
+
+      job = ShopifyApp.configuration.after_authenticate_job[:job]
```

Comment:
Is it possible for `ShopifyApp.configuration.after_authenticate_job` to be truthy, without setting `ShopifyApp.configuration.after_authenticate_job[:job]` ?